SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -225,6 +227,59 @@ namespace ESpatialOSNetFlow
 	};
 }
 
+UENUM()
+namespace ESpatialOSRuntimeVariant
+{
+	enum Type
+	{
+		Standard,
+		CompatibilityMode
+	};
+}
+
+USTRUCT()
+struct SPATIALGDKEDITOR_API FRuntimeVariantVersion
+{
+	friend class USpatialGDKEditorSettings;
+	friend class FSpatialRuntimeVersionCustomization;
+
+	GENERATED_BODY()
+
+	FRuntimeVariantVersion();
+
+	FRuntimeVariantVersion(const FString& InPinnedVersion) : PinnedVersion(InPinnedVersion)
```

Comment:
Do we need this constructor?

Reply:
Yeah - it's initialised here 
![image](https://user-images.githubusercontent.com/31517089/83274154-4b2b9c80-a1c5-11ea-8bbc-aa44c796ee06.png)

Reply:
Do we need the default constructor then?

Reply:
UHT will complain otherwise it seems